client/consensus/babe/src/rpc.rs

```diff
@@ -0,0 +1,165 @@
+// Copyright 2019-2020 Parity Technologies (UK) Ltd.
```

Comment:
This should all be in `client/rpc`.

Reply:
hmm, I thought `client/rpc` was for core code. Since consensus is pluggable, it wouldn't make sense for this to be in there.

maybe @rphmeier can clarify.

Reply:
That is okay. However, the copyright is not correct:
```suggestion
// Copyright 2020 Parity Technologies (UK) Ltd.
```